dist/css/windows.css

```diff
@@ -1,239 +1,424 @@
-        font-weight: 500;
-        letter-spacing: 1.5px ;
-    }
-    #floatingWindows > .windowFrame > .wfWinBody {
-        flex-direction: column;
-        flex-grow: 1;
-    }
-    #floatingWindows > .windowFrame > .wfWinBody > .wfbToolbar {
-        width: 54px;
-        flex-shrink: 0;
-        display: flex;
-        overflow-y: auto;
-        background-color: rgba(0, 0, 0, 0.1);
-        width: 100%;
-        padding: 2px 4px;
-    }
-    #floatingWindows > .windowFrame > .wfWinBody > .wfbToolbar > * {
-        flex-shrink: 0;
-    }
-    #floatingWindows > .windowFrame > .wfWinBody > .wfbWidget {
-        padding: 10px;
-    }
+  .wheelNav {
+    z-index: 2000;
+  }
+
+  #floatingWindows {
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: 10000;
+  }
+
+  #floatingWindows > .windowFrame {
+    border-radius: 0;
+    border: none;
+    width: 100vw;
+    height: 100vh;
+    left: 0 !important;
+    top: 0 !important;
+    background-color: #e8e8e8;
+    display: flex;
+    flex-direction: column;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar {
+    height: 56px;
+    background-color: #2196f3;
+    border: none;
+    padding: 0;
+    align-items: center;
+    flex-direction: row-reverse;
+    flex-shrink: 0;
+    box-shadow: 0px 2px 5px 0px rgba(0, 0, 0, 0.16), 0px 2px 10px 0px rgba(0, 0, 0, 0.12);
+    min-width: auto;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton {
+    padding: 0 15px;
+    background: none;
+    border-radius: 0;
+    transition: background-color 300ms;
+    height: 100%;
+    width: 56px;
+    margin: 0;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton.wftMaxmin {
+    display: none;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton.rollup {
+    display: none;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton:hover {
+    opacity: 1;
+    background-color: rgba(0, 0, 0, 0.1);
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton.rollup::before {
+    width: 40%;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton.close::before,
+  #floatingWindows > .windowFrame > .wfTopBar .wftButton.close::after {
+    width: 50%;
+  }
+
+  #floatingWindows > .windowFrame > .wfTopBar .wftTitle {
+    flex-grow: 1;
+    color: #fff;
+    font-size: 1.5em;
+    padding: 0 15px;
+    text-align: center;
+    text-transform: uppercase;
+    font-weight: 500;
+    letter-spacing: 1.5px;
+  }
+
+  #floatingWindows > .windowFrame > .wfWinBody {
+    flex-direction: column;
+    flex-grow: 1;
+  }
+
+  #floatingWindows > .windowFrame > .wfWinBody > .wfbToolbar {
+    width: 54px;
+    flex-shrink: 0;
+    display: flex;
+    overflow-y: auto;
+    background-color: rgba(0, 0, 0, 0.1);
+    width: 100%;
+    padding: 2px 4px;
+  }
+
+  #floatingWindows > .windowFrame > .wfWinBody > .wfbToolbar > * {
+    flex-shrink: 0;
+  }
+
+  #floatingWindows > .windowFrame > .wfWinBody > .wfbWidget {
+    padding: 10px;
+  }
+}
+
```

Comment:
From this line, I have added media queries for responsiveness in different devices